landscape/categories.md

```diff
@@ -23,24 +23,29 @@ sub-categories. The remaining text is a description of each sub-category.
   - _Secret Management_
     - Tools that help to manage, encrypt or distribute secrets and other sensitive information.
 
-- **Identity & Access Control**
-  - _Identity_
-    - SPIFFE, identity providers, OpenID, LDAP, Okta
+- **Identity Lifecycle and & Access Management**
+  - _Identity Lifecycle_
+    - Systems for creating, governing and terminating identities and entitlements for users and systems
+      - Lifecycle management tools like Okta, FreeIPA
```

Comment:
Do we want to name specific things here? Is this a governing document describing the category, or the root of discussion?

Reply:
Taking from my comment in https://github.com/cncf/sig-security/issues/234, I'm not a fan of the naming of this being restricted to the particular activities, but I think putting it down like you've done here is good. Perhaps if we presented it as identity related activities including x, y, z.